### ISSUE TITLE
Norwegian Meteorological Institute weather forecast not updating

### ISSUE THREAD

**Issue by johnno66** [blackadderkate/weather-widget-2#150]
Since a few days the widget (2.3.4) does no longer receive updates from the Norwegian Meteorological Institute.
OpenWeather still updates fine.

Added a new location and tested api.met.no is reachable (it is) but no weather data unfortunately  :-(

**Comment by blackadderkate** [comment]
Yes, this happened to me too - I have a potential fix here: [v2.3.5 beta](https://github.com/blackadderkate/weather-widget-2/releases/download/v2.3.5beta/weather-widget-2.3.5.plasmoid) please can you download it and let me know if it fixes the problem?

**Comment by bob23450** [comment]
Same issue here, for me v2.3.5 beta above fixed the problem.
Thank you

**Comment by johnno66** [comment]
v.2.3.5 fixed the issue.
Thanks for the quick resolution!

**Comment by pstrejajo** [comment]
As only accepting status 203 is not a proper solution per se for this case (or at least a final fix), I have created a PR #151 which switches to Sunrise 3.0 API.

**Comment by blackadderkate** [comment]
OK, uploaded a new version of the widget with a much better fix submitted by @pstrejajo .

Find it here: [2.3.5 Beta 2](https://github.com/blackadderkate/weather-widget-2/releases/download/v2.3.5beta2/weather-widget-2.3.5.plasmoid)

**Comment by SergMSerg** [comment]
hello friends, I faced same problem in last year may be since I moved from Debian to Kubuntu.
I found it on my laptop first and IMHO the problem related to WiFi connection, problem appears after sleep/wakeup and NOT appears right after boot/reboot; moreover, other weather widgets act same way (wunderground weather for example), version 2.3.5 has same problem;
meanwhile is there a way to quick restart a widget in on click (or better with command line)?
for now I can restart KDE session and widget works again but this is too rough way, disconnect WiFi and connect it back does not help

**Comment by solbu** [comment]
> meanwhile is there a way to quick restart a widget in one click (or better with command line)? for now I can restart KDE session and widget works again but this is too rough way

I run this command, and the desktop _environment_ is restarted (restarts the start menu, taskbar and all widgets), but all regular programs (browser, editors, terminals etc) keeps running.
`kquitapp5 plasmashell && kstart5 plasmashell`

**Comment by SergMSerg** [comment]
wow, it worked! widget restarted and updated successfully, thankk you!

**Comment by GioBego** [comment]
Can this be pushed from beta to full release?

**Comment by solbu** [comment]
For about 3 days, this started to affect me too.
A new release is needed.

**Comment by solbu** [comment]
I can also confirm that the beta2 works, I just packaged and installed it on my local systems. :-)
